SwiftAudio/Classes/AudioPlayer.swift

```diff
@@ -158,10 +158,11 @@ public class AudioPlayer: AVPlayerWrapperDelegate {
             try self.wrapper.load(fromFilePath: item.getSourceUrl(), playWhenReady: playWhenReady)
         }
         
+        wrapper.currentItem?.audioTimePitchAlgorithm = item.getPitchAlgorithmType() as String
```

Comment:
This conversion to String triggers an error in Travis:
`/Users/travis/build/jorgenhenrichsen/SwiftAudio/SwiftAudio/Classes/AudioPlayer.swift:161:61: cannot convert value of type 'AVAudioTimePitchAlgorithm' to type 'String' in coercion`

Seems like the conversion to String is not valid to do here. I think you can just use the value directly?

Reply:
this is the part that I don't get. This is the only way it compiles on my side.. 

let me look into it a bit further

Reply:
I see, what is the error you get when just using the value directly?